src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Symbols/SymbolManagerBase.cs

```diff
@@ -337,16 +331,18 @@ public bool Equals(TypeArrayKey other)
                 if (other._types == _types) return true;
                 if (other._types.Length != _types.Length) return false;
                 if (other._hashCode != _hashCode) return false;
-                for (int i = 0, n = _types.Length; i < n; i++)
+                for (int i = 0; i < _types.Length; i++)
```

Comment:
No, I'm sure it will. I started a fuller change that would have pulled it local too, then decided to leave that until later, but didn't undo what I'd done fully resulting in the worse of both worlds. I'll rectify that soon.